src/platform-implementation-js/dom-driver/gmail/views/gmail-collapsible-section-view.ts

```diff
@@ -625,7 +638,7 @@ class GmailCollapsibleSectionView {
   #addToCollapsedContainer() {
     const element = this.#element;
     if (!element) return;
-    if (this.#headerElement) this.#headerElement.classList.remove('Wg');
+    // if (this.#headerElement) this.#headerElement.classList.remove('Wg');
```

Comment:
should this just be deleted? if not a comment to say why?

Reply:
Removed.